walletdatabase/migrations/sql/1715839555_rename_chain_prefixes.up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE saved_addresses SET chain_short_names = REPLACE(chain_short_names, 'arb:', 'arb1:') WHERE chain_short_names LIKE '%arb:%';
```

Comment:
@smohamedjavid could you confirm that `REPLACE` works correctly on iOS?

Reply:
Yep, It's all good 👍